vcpkg/maintainers/handling-usage-files.md

```diff
@@ -29,18 +29,18 @@ While it can be tempting to include extensive code snippets, command-line instru
 
 For more comprehensive and detailed information, users can refer to the [documentation](../users/manifests.md).
 
-The usage file should follow this template:
+We ask that usage files follow a similar pattern to the templates below.
```

Comment:
CPub style is to do something like this:

Usage files should follow a pattern similar to the following templates: